### ISSUE TITLE
Create a tox.ini file

### ISSUE THREAD

**Issue by camilonova** [jazzband/sorl-thumbnail#525]
I guess is better than having all at the `.travis.yml` file.

Can anyone please work on this? I'll be happy to review and merge it.

**Comment by camilonova** [comment]
@WhyNotHugo or @Flimm maybe?

**Comment by haakenlid** [comment]
There's a `[tox]` section in `setup.cfg` as well.

**Comment by camilonova** [comment]
Seems is a better practice to have a tox.ini file

**Comment by WhyNotHugo** [comment]
Given that having a `tox.ini` file seems to be the norm, I'd be okay with spiting that out from `setup.cfg`. I had a hard time finding where tox's config was for this repo.

**Comment by Flimm** [comment]
**tldr;** There's no common best practise about whether to use `tox.ini` or `setup.cfg`, so let's use `tox.ini`.

---

I need to point out that if Tox configuration is in `setup.cfg`, it should be under `[tox:tox]`, not `[tox]` (as it is now). I'm not sure why it seems to work in the latter case sometimes. I have reported the fact that `[tox]` in `setup.cfg` without a prefix works sometimes here: https://github.com/tox-dev/tox/issues/545

I'm not sure there is a best practise in the Python world of whether to use `setup.cfg` or `tox.ini` for Tox configuration. The Tox project seemed to encourage using only `tox.ini` in the past, with `setup.cfg` being an undocumented feature. However, there seems to be a movement among some Python developers to start moving more and more configuration to `setup.cfg` instead of increasing the number of files. For instance, Guido Von Rossum was persuaded to add support for Tox configuration in the `setup.cfg` file in this pull request for the mypy project: https://github.com/python/mypy/pull/2761#issuecomment-279305222

Let's move the configuration to `tox.ini` anyway, since this what our developers (@camilonova, @WhyNotHugo) seem to prefer.
